redbot/core/commands/help.py

```diff
@@ -390,9 +401,9 @@ def shorten_line(a_line: str) -> str:
                 )
                 for i, page in enumerate(pagify(subtext, page_length=500, shorten_by=0)):
                     if i == 0:
-                        title = _("**__Subcommands:__**")
+                        title = f"{bold(underline(_('Subcommands:')))}"
                     else:
-                        title = _("**__Subcommands:__** (continued)")
+                        title = f"{bold(underline(_('Subcommands: (continued)')))}"
```

Comment:
The ones like this one which are just `f"{...}"` (one formatted part with nothing unformatted) don't need to be fstrings
```suggestion
                        title = bold(underline(_("Subcommands: (continued)")))
```
(Quotes changed because I expect black to yell if you don't :P)